credsweeper/app.py

```diff
@@ -73,15 +76,28 @@ def __init__(self,
         self.json_filename: Optional[str] = json_filename
         self.ml_batch_size = ml_batch_size
         self.ml_threshold = ml_threshold
-        if self._use_ml_validation():
-            from credsweeper.ml_model import MlValidator
-            self.ml_validator = MlValidator(threshold=self.ml_threshold)
+        # the import cannot be done on top due
+        # TypeError: cannot pickle 'onnxruntime.capi.onnxruntime_pybind11_state.InferenceSession' object
+        from credsweeper.ml_model import MlValidator
+        self.__ml_validator: Any[MlValidator] = None
```

Comment:
Thank you for adding `@property` here.
But I think it is good to access `self.__ml_validator` attribute by using `@property` like as `self.ml_validator`.. :thinking: 
How do you think about it?

Reply:
There is an issue with 'Any'...
Probably if we init MlValidator in Credswepeer constructor - it possible.

Reply:
in case of the setter using mypy warns in the getter:
> credsweeper/app.py:95: error: Cannot determine type of "__ml_validator"

Without 'Any' we got serialization error.
:(

Reply:
Done. Now all checks are satisfied.